src/settings/components/RoleDetails/RoleDetails.js

```diff
@@ -92,6 +101,15 @@ const RoleDetails = ({ onClose, roleId }) => {
           <AccordionUsers roleId={roleId} />
         </AccordionSet>
       </AccordionStatus>
+      <ConfirmationModal
+        open={isDeleting}
+        heading={<FormattedMessage id="ui-authorization-roles.crud.deleteRole" />}
+        message={<><FormattedMessage id="ui-authorization-roles.crud.deleteRoleConfirmation" values={{ rolename: role?.name }} />
+          {' '} <b><FormattedMessage id="ui-authorization-roles.crud.deleted" /></b> </>}
```

Comment:
Please embed the styling in the translation to avoid breaking a single phrase across multiple messages.